builtin/providers/google/resource_compute_instance_group_test.go

```diff
@@ -14,19 +14,32 @@ import (
 func TestAccComputeInstanceGroup_basic(t *testing.T) {
 	var instanceGroup compute.InstanceGroup
 	var instanceName = fmt.Sprintf("instancegroup-test-%s", acctest.RandString(10))
+	var networkName = "default"
+	projs := []string{
+		"GOOGLE_PROJECT",
+		"GCLOUD_PROJECT",
+		"CLOUDSDK_CORE_PROJECT",
+	}
+	var networkUrl = fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s", multiEnvSearch(projs), networkName)
```

Comment:
This test doesn't feel very basic anymore. Mind creating a new test for this? I also think creating the network from within the test would be nicer than relying on the default network and having to read the project from the environment.